### ISSUE TITLE
Agenda for PM CoP roadmap meetings

### ISSUE THREAD

**Issue by ebele-oputa** [hackforla/product-management#319]
### Overview
This issue tracks the agenda for the PM CoP roadmap meetings.
11/11/21

### Resources
#309

**Comment by ebele-oputa** [comment]
Agenda for 11/11/21

- Roll call
- Research updates
-

**Comment by ebele-oputa** [comment]
> Agenda for 11/11/21
> 
> * Roll call
> * Research updates [Create a roadmap for PM COP #309](https://github.com/hackforla/product-management/issues/309)
> * Review of PM responsibilities doc [Create a responsibilities doc for CoP #308](https://github.com/hackforla/product-management/issues/308)
> * PM CoP Google File Migration [PM CoP Google File Migration #313](https://github.com/hackforla/product-management/issues/313)

### Research Updates
The assigned UX Researcher is reviewing the documents/issues and will produce a list of findings and/or a research plan for further studies

**Comment by ExperimentsInHonesty** [comment]
- [ ] Update slack reminders that contain the zoom link to the CoP meetings

**Comment by ExperimentsInHonesty** [comment]
11/18/21

* Research updates [Create a roadmap for PM COP #309](https://github.com/hackforla/product-management/issues/309)
* Review of PM responsibilities doc [Create a responsibilities doc for CoP #308](https://github.com/hackforla/product-management/issues/308)
* PM CoP Google File Migration [PM CoP Google File Migration #313](https://github.com/hackforla/product-management/issues/313)

**Comment by ebele-oputa** [comment]
12/02/21 Meeting

Roll call - Bonnie, Olivia, Ebele

Decisions:
 - Modify the roadmap meeting time to suit the availability of the PM CoP leads
 - Postpone the product management community of practice meeting till January
 - Provide progress updates during the next meeting

**Comment by ExperimentsInHonesty** [comment]
Next meeting discuss this
- https://github.com/hackforla/product-management/issues/259

**Comment by priyatalwar** [comment]
Add this issue to this week's agenda: https://github.com/hackforla/UI-UX/issues/189

**Comment by priyatalwar** [comment]
## CoP Meeting Agenda 2022-1-5

### prework

### Roll Call:


### Recurring Action Items
- [x]  Accountability and Support Check.
  - [ ]  Review assignments for each CoP Lead (starting with those present)
     - [ ] [Bonnie](https://github.com/hackforla/product-management/issues/assigned/ExperimentsInHonesty)
     - [ ] [Priyanka](https://github.com/hackforla/product-management/issues/assigned/priyatalwar)
     -  [ ] [Ebele](https://github.com/hackforla/product-management/issues/assigned/ExperimentsInHonesty)
     - [ ] [Carol](https://github.com/hackforla/product-management/issues/assigned/Rabia2219)

### Other agenda items
- [ ] Roadmap



### Notes from meeting

**Comment by priyatalwar** [comment]
[DATE] Agenda and Notes

### Prework
- [ ] issue for audit #93

### Roll Call
- [ ] Bonnie
- [ ] Rabia
- [ ] Priyanka

#### Recurring items: Happens on the PM Monday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/internship/projects/1#column-15516595)
 - [ ] check Branding issue to see if there is anything the team needs to do, so that we can prioritize it #68
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Priyanka](https://github.com/hackforla/internship/issues/assigned/priyatalwar)
       - [ ] [Rabia](https://github.com/hackforla/internship/issues/assigned/Rabia2219)
       - [ ] [Seena](https://github.com/hackforla/internship/issues/assigned/seenaiype)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/internship/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review the [All team meeting Agenda](https://github.com/hackforla/internship/issues/101) and make it ready for the next team meeting

### Items Carried Forward

#### New Items

  - [ ] 
  - [ ] 
  - [ ] Carry forward the "New Items" that we did not get to look at today
  
### FYIs

#### Task items from Team meeting to review
- [ ] 
- [ ] 

### Notes from meeting
- 

### New Task Items

- [ ] 

#### Items for next week's agenda

- [ ]

**Comment by chiusano** [comment]
Mike here from the Marketing Team.  Bonnie said at next CoP meeting she will speak to PMs regarding video project submissions and receiving a response as to whether or not they will participate.

[github issue](https://github.com/hackforla/marketing/issues/54)

**Comment by ExperimentsInHonesty** [comment]
Let's add this to a brain storming meeting https://github.com/hackforla/product-management/issues/259  Also, asking each pm to identify the constraints for one tool at each meeting.

**Comment by ExperimentsInHonesty** [comment]
Usefull items in the CoP
- Bryce, HUU
  - Getting to know the other PMs
  - Understanding what else is happening at Hack for LA (org wise)
  -

**Comment by ExperimentsInHonesty** [comment]
Areas for improvement
- more sharing of examples
- Agenda varies, potentially more structure

**Comment by ExperimentsInHonesty** [comment]
# 2023-08-04

- [ ] Review the engineering CoP Meeting schedule format https://github.com/hackforla/engineering/wiki/CoP-Meetings

**Comment by joyceguo99** [comment]
Action items:
- [ ] In Onboarding Deck, slide 35, add team specific guide to create experience ticket
- [ ] Move everyone's experience ticket to appropriate project board
- [ ] Recruit for other CoP leads for the PM community of practice
- [ ] Figure out when to meet for planning

**Comment by 1610saas** [comment]
### 12/01/23 Meeting - draft

**Comment by ExperimentsInHonesty** [comment]
I can be available at 1 of the pm meetings per month to answer HfLA questions.  Anyone who has questions could add it to an issue with each question being a comment, and then we can add answers to the comments, for other people to read.

**Comment by ExperimentsInHonesty** [comment]
Ksenia reached out to the other peoples who are interested in co-leading and is waiting to hear back from them

**Comment by dolla24** [comment]
**1/5/23 Meeting**
**Discussed**:

We may likely change our meeting schedule dates (weekdays likely 4:00 -7:00)
Types of activities we want to include
Workshops - subject and industry specific (**Workshop review about assigning user stories, effort estimation, acceptance criteria)**
Networking - Happy Hour; Getting to know the other PMs **TBD**
Meet a "Speaker" - external or internal speaker sharing personal professional experience; QA session included. **(In the works a speaker with verse industry experience "FreedomPop", Internet Brands", "DISH Network"**

We talked about opening a Q&A for new members on any topic of inquiries.

**Comment by ExperimentsInHonesty** [comment]
## 2025-01-29 Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
Notes from 2025-01-29
- We reviewed issue #7 and it was not applicable to the onboarding and offboard process for PMs on CoPs

**Comment by Kphalguni** [comment]
Notes from 2025-02-11
- We added and removed from 1Password and VRMS
- We added all Weekly Label Check issue that PMs provided
- Reviewed the next agenda 2025-02-14

### Tasks
- [ ] Add the remaining the Weekly Label Check from Bonnie's comment to the template and update the 2025-02-14 agenda
- [ ] Add calendar filter to the product-management to the email address
   - https://github.com/hackforla/product-management/wiki/Create-filters-to-move-calendar-invites-to-folders

**Comment by ExperimentsInHonesty** [comment]
Tasks
- [ ] Make a weekly label check issue from this template https://github.com/hackforla/product-management/wiki/Weekly-Label-Check
   - [ ] add reviewing it to the agenda

**Comment by Kphalguni** [comment]
## 2025-02-17 Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #


### Items from last meeting that did not get completed
- [ ] Finish Onboarding Phalguni https://github.com/hackforla/product-management/issues/387#issuecomment-2622508504

### New Items
- Note-Taking
   - [ ] talk about how we want to manage note-taking for the following meetings
      - [ ] planning meeting (this one)
      - [ ] the Friday meeting
      - [ ] How to start the Friday meeting
- [ ] Roadmap
   - [ ] review task items
   - [ ] Review issues Bonnie is assigned to
   - [ ] clean up the board (remove people that are no longer active, from their issues and put their issues back into new issue approval.
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

Notes from 2025-02-11
- We added and removed from 1Password and VRMS
- We added all Weekly Label Check issue that PMs provided
- Reviewed the next agenda 2025-02-14

### Task Items
- [x] Add the remaining the Weekly Label Check from Bonnie's comment to the template and update the 2025-02-14 agenda
- [ ] Add calendar filter to the product-management to the email address
   - https://github.com/hackforla/product-management/wiki/Create-filters-to-move-calendar-invites-to-folders

**Comment by ExperimentsInHonesty** [comment]
I am 100% certain that the board is in need of a big cleanup, so currently there is no point in looking at the new issue approval column.  We will do that after we get on top of the following
- create and perform the weekly label check
- move things on the board, that are stale or unclear
-

**Comment by ExperimentsInHonesty** [comment]
Add this to next agenda https://github.com/hackforla/admin/issues/93#issuecomment-2669540596

**Comment by Kphalguni** [comment]
2025-02-25 Meeting Agenda
Prework to prep for meeting

#
Recurring items:

review any issues that are in the new issue approval column [CoP: PM: Project Board](https://github.com/orgs/hackforla/projects/85/views/1)

Accountability and Support Check.

Review assignments for each Team member

[Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)

[Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)

[Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)

finish adding labels to the issue that are still missing them (see audit links above)

review issues with [epic label] and define how to break down further

Review to see if there are any members to Onboard or Offboard
- [ ] #
Items from last meeting that did not get completed

Finish Onboarding Phalguni https://github.com/hackforla/product-management/issues/387#issuecomment-2622508504
New Items
Note-Taking

talk about how we want to manage note-taking for the following meetings

planning meeting (this one)

the Friday meeting

How to start the Friday meeting

Roadmap

review task items

Review issues Bonnie is assigned to

clean up the board (remove people that are no longer active, from their issues and put their issues back into new issue approval.

Carry over Items to next meeting that don't get addressed
FYIs
Notes from Meeting
Task Items

Add the remaining the Weekly Label Check from Bonnie's comment to the template and update the 2025-02-14 agenda

Add calendar filter to the product-management to the email address
https://github.com/hackforla/product-management/wiki/Create-filters-to-move-calendar-invites-to-folders

Make a weekly label check issue from this template https://github.com/hackforla/product-management/wiki/Weekly-Label-Check

add reviewing it to the agenda

**Comment by Rabia2219** [comment]
#### PM CoP Rosters

- [product-management-team-roster (current)](https://docs.google.com/spreadsheets/d/1mDMUAZchhpCSMU9D9i5jkQU172_90cZ9rr_5l-4rZlM/edit?gid=0#gid=0)
- [Product Management team roster]( https://docs.google.com/spreadsheets/d/1SBdwrwBa-XIV2LCLha4DDL5r_-VN5FPu82moVVbgYcY/edit?gid=1806338047#gid=1806338047)

**Comment by ExperimentsInHonesty** [comment]
- https://github.com/hackforla/product-management/issues/388

**Comment by Kphalguni** [comment]
## [2025-03-04]  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  Add Vihar to TDM https://github.com/hackforla/product-management/issues/386#issuecomment-2691525139

### Agenda Items

#### items carried over from prior meetings
- [ ] Finish Onboarding Phalguni https://github.com/hackforla/product-management/issues/387#issuecomment-2622508504
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Roadmap
   - [ ] review task items
   - [ ] Review issues Bonnie is assigned to
   - [ ] clean up the board (remove people that are no longer active, from their issues and put their issues back into new issue approval.
- [ ] Onboarding deck - Pending Bonnie's Review: https://github.com/hackforla/admin/issues/93#issuecomment-2669540596
- [ ] Carry over Items to next meeting that don't get addressed

#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings
- [ ] Cleanup the Google Drive
- [ ] Add calendar filter to the product-management to the email address
   - https://github.com/hackforla/product-management/wiki/Create-filters-to-move-calendar-invites-to-folders
- [ ] Populate the roster with each person's name and team assignment from the https://github.com/hackforla/product-management/issues/386
   - [ ] We need to add everyone from write to read GitHub teams, and then remove people from write access if they are not a pm on a team

#### new task items

**Comment by Kphalguni** [comment]
## [2025-03-10]  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] Finish Onboarding Phalguni https://github.com/hackforla/product-management/issues/387#issuecomment-2622508504
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Roadmap
   - [ ] review task items
   - [ ] Review issues Bonnie is assigned to
   - [ ] clean up the board (remove people that are no longer active, from their issues and put their issues back into new issue approval.
- [ ] Onboarding deck - Pending Bonnie's Review: https://github.com/hackforla/admin/issues/93#issuecomment-2669540596
- [ ] Carry over Items to next meeting that don't get addressed
#### new items
- [ ] Carry over Items to next meeting that don't get addressed


### FYIs
-

### Notes from Meeting

### Task Items

- [ ] Cleanup the Google Drive
- [ ] Add calendar filter to the product-management to the email address
   - https://github.com/hackforla/product-management/wiki/Create-filters-to-move-calendar-invites-to-folders
- [ ] Populate the roster with each person's name and team assignment from the https://github.com/hackforla/product-management/issues/386
   - [ ] We need to add everyone from write to read GitHub teams, and then remove people from write access if they are not a pm on a team
#### unfinished tasks from prior meetings

#### new task items

**Comment by Kphalguni** [comment]
## [2025-03-18]  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Roadmap
   - [ ] review task items
   - [ ] Review issues Bonnie is assigned to
   - [ ] clean up the board (remove people that are no longer active, from their issues and put their issues back into new issue approval.
#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] We need to add everyone from write to read GitHub teams, and then remove people from write access if they are not a pm on a team

#### new task items

**Comment by Kphalguni** [comment]
## [2025-03-25]  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Roadmap
   - [ ] review task items
   - [ ] Review issues Bonnie is assigned to
   - [ ] clean up the board (remove people that are no longer active, from their issues and put their issues back into new issue approval.
#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team

#### new task items

**Comment by Kphalguni** [comment]
## [2025-04-01]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  Remove Jasika from TDM: https://github.com/hackforla/product-management/issues/386#issuecomment-2762218885

### Agenda Items

#### items carried over from prior meetings

- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Roadmap
   - [ ] review task items
   - [ ] Review issues Bonnie is assigned to
   - [ ] clean up the board (remove people that are no longer active, from their issues and put their issues back into new issue approval.
#### new items
- [ ] Carry over Items to next meeting that don't get addressed

#### new items

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team
         
#### new task items

**Comment by Kphalguni** [comment]
## [2025-04-07]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Roadmap
   - [ ] review task items
   - [ ] Review issues Bonnie is assigned to
   - [ ] clean up the board (remove people that are no longer active, from their issues and put their issues back into new issue approval.
#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team
#### new task items

**Comment by Kphalguni** [comment]
## [2025-04-15]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team
#### new task items

**Comment by Kphalguni** [comment]
## [2025-04-22]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team
#### unfinished tasks from prior meetings

#### new task items

**Comment by Kphalguni** [comment]
## [2025-04-26]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team
#### unfinished tasks from prior meetings

#### new task items

**Comment by Kphalguni** [comment]
## [2025-05-06]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team
#### unfinished tasks from prior meetings
#### unfinished tasks from prior meetings

#### new task items

**Comment by ExperimentsInHonesty** [comment]
Agile
Behavior-Driven Development (tests before writing the code)
DDD (Domain-Driven Development)
FDD (Feature-Driven Development)
TDD (Test-Driven Development)
XP (Extreme Programming)
BDD (Behavior Driven Development)
Lean

**Comment by ExperimentsInHonesty** [comment]
Meetup post

```
🧠 Talk Title: How We Use Agile Methodologies and Practices at Hack for LA
📅 Date: [Insert Date]
🕕 Time: [Insert Time]
📍 Location: [Insert Venue or "Online Event"]
🎟 RSVP: [Insert RSVP link or instructions]

🚀 About the Talk
Curious how Agile really works in the wild? At Hack for LA, we apply Agile not just by the book, but in ways that meet the unique demands of civic tech projects — with remote teams, rotating volunteers, and constantly evolving needs.

In this session, we’ll share:

A quick overview of Agile’s origins, values, and principles

How we’ve adapted Scrum ceremonies to fit our workflow

When and why we use Kanban and Lean to manage flow and reduce waste

Real-world examples of how we use practices like

TDD (Test-Driven Development)

BDD (Behavior-Driven Development)

DDD (Domain-Driven Design)

FDD (Feature-Driven Development)

XP (Extreme Programming)

Whether you’re new to Agile or looking to see how its practices can flex to serve mission-driven work, this talk is for you.

🎯 Who Should Attend
Developers, designers, and product folks using (or curious about) Agile

Project managers and tech leads

Volunteers in civic tech or open source

Anyone interested in applying structured collaboration to complex work

🗣 Speaker
[Your Name] – [Your role at Hack for LA or relevant background]

📝 RSVP Now
[Insert RSVP link again]

Bring your questions and curiosity. We’ll leave time for discussion, Q&A, and maybe even a few Agile war stories.

Let me know if you'd like a shorter version for Twitter/X or a banner-style event description.
```

**Comment by ExperimentsInHonesty** [comment]
### For Twitter/X (with hashtags):
```
How do civic tech teams use Agile IRL?
Join us as Hack for LA shares how we apply Scrum, Kanban, TDD, BDD & more on real projects.
📅 [Date] 🕕 [Time] 📍 [Online]
🔗 [short RSVP link]
#Agile #CivicTech #Scrum #TDD #HackForLA #TechForGood
`

### For LinkedIn (with hashtags):
```
🚀 How does Agile actually work in civic tech?
At Hack for LA, we’ve adapted Scrum, Kanban, TDD, BDD, and more to fit real-world volunteer projects.
Join us for a practical talk on Agile in action!
📅 [Date] | 🕕 [Time] | 📍 [Online]
👉 RSVP: [Link]
#Agile #Scrum #Kanban #TDD #BDD #CivicTech #HackForLA #TechForGood #ProjectManagement
```

### Instagram (caption + hashtags)
```
👩‍💻 Ever wonder how Agile really works on mission-driven projects?

Join us as Hack for LA shares how we use Scrum, Kanban, TDD, BDD, and more to power civic tech work that makes a difference.

📅 [Date]
🕕 [Time]
📍 [Online or Location]
🔗 Link in bio or below to RSVP!

#Agile #Scrum #Kanban #TDD #BDD #CivicTech #TechForGood #HackForLA #VolunteerTech #ProjectManagement #ProductDevelopment #OpenSource
```

### Threads
```
How do civic tech teams use Agile?
Hack for LA will show how we’ve adapted Scrum, Kanban, TDD, BDD, and more to build public-good projects that work.
📅 [Date] 🕕 [Time]
RSVP 👉 [link]

#Agile #CivicTech #HackForLA #TechForGood #Scrum #Kanban #BDD #TDD #OpenSource
```

**Comment by ExperimentsInHonesty** [comment]
Palette https://coolors.co/palette/669900-99cc33-ccee66-006699-3399cc-990066-cc3399-ff6600-ff9900-ffcc00

**Comment by Kphalguni** [comment]
## [2025-05-13]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
- Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team
   
#### unfinished tasks from prior meetings

#### new task items

**Comment by Kphalguni** [comment]
## [2025-05-20]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Zoom background for HfLA projects
- [ ] Remove AI notetakers from them meeting invites

### FYIs
-

### Notes from Meeting

### Task Items
- [ ] Note-Taking
- [ ] talk about how we want to manage note-taking for the following meetings
   - [ ] planning meeting (this one)
- [ ] Cleanup the Google Drive
   - [ ] organize files
   - [ ] remove access and make sure that all current product managers have access
- [ ] Cleanup GitHub using sheet 1[Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg) and sheet 2 [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - [x] Go to Sheet 1 Roster and grab an email address and search it in the sheet 2 and if there add TRUE into sheet 1 & 2
      - [x] If its not there, search that same person by name in sheet 2, if they definitely are not there, mark it as FALSE in sheet 1
         - [ ] [HOLD - waiting on the above and more to be done] add everyone from write who has been a PM to Product-Management GitHub team
         - [ ]  [HOLD - waiting on the above and more to be done] remove people from Product-Management-write team access if they are not a pm on a team

#### unfinished tasks from prior meetings

#### new task items

**Comment by Kphalguni** [comment]
## [2025-05-27]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Provide roster access to Phalguni 
- [ ] Remove AI notetakers from them meeting invites

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings

#### new task items

**Comment by Kphalguni** [comment]
1 password > HFLA-email-all-vault- to access all roster

**Comment by Kphalguni** [comment]
## [2025-06-10]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings

#### new task items

**Comment by Kphalguni** [comment]
## [YYYY-MM-DD]  Meeting Agenda

### Prework to prep for meeting
- [ ] https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer
- [ ] https://github.com/hackforla/product-management/issues/388

### Recurring items: 
 - [ ] review any issues that are in the new issue approval column [CoP: PM: Project Board
](https://github.com/orgs/hackforla/projects/85/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AExperimentsInHonesty)
       - [ ] [Phalguni](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3AKphalguni)
       - [ ] [Rabia](https://github.com/hackforla/product-management/issues?q=is%3Aissue%20state%3Aopen%20assignee%3ARabia2219)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### Agenda Items

#### items carried over from prior meetings
- [ ] 

#### new items
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

#### unfinished tasks from prior meetings

#### new task items

**Comment by Kphalguni** [comment]
- [ ] Linkedin is messed up from Row number 193 to 226  [Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg)
- [ ] This was copied from the original link from HFLA website team roster where the linkedIn info is not matching against name [Hack for LA Website Team Roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit?gid=0#gid=0)

**Comment by Kphalguni** [comment]
@ExperimentsInHonesty and @Rabia2219 

**Changes made after our call in** [Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg)
- Copied from VRMS team Roster:
   - Added 7 more names (row number 292–298) from the VRMS team roster to [Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg). Also verified these against [CoP: Product: All people who have been added to GitHub write](https://docs.google.com/spreadsheets/d/1WS845bP9YXiq5qVwd0rYRIfVIgn8O6atqN4oGPRLUes)
   - I had kept this pending because the original VRMS roster link in the weekly CoP agenda issue was incorrect. I now retrieved the correct roster from the Drive folder (hfla-email-all) and also updated the link in the weekly CoP PM agenda.

- No action was taken from the following CoPs' team rosters, as they either had no PM names listed or would have resulted in duplicates of existing entries
   - CoP PM
   - CoP: Data Science
   - CoP: DevOps
 
- No team roster link available for CTI in the CoP PM weekly agenda

**Comment by Rabia2219** [comment]
- move the AISA meeting to tonight 7 pm PST
